go/private/repositories.bzl

```
@@ -82,6 +82,25 @@ def go_rules_dependencies(force = False):
         patch_args = ["-p1"],
     )
 
+    # Needed for go/tools/fetch_repo
+    # releaser:upgrade-dep golang tools
+    wrapper(
+        http_archive,
+        name = "org_golang_x_tools_go_vcs",
+        # v0.12.0, latest as of 2023-08-12
+        urls = [
+            #"https://mirror.bazel.build/github.com/golang/tools/archive/refs/tags/go/vcs/v0.1.0-deprecated.zip",
```

```suggestion
          "https://mirror.bazel.build/github.com/golang/tools/archive/refs/tags/go/vcs/v0.1.0-deprecated.zip",
```

after https://github.com/bazelbuild/bazel/issues/19335 has been resolved.

---

Thanks for noticing and requesting the file to be mirrored. Totally forgot about that!